### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by noniagriconomie**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| License       | MIT

Again small improvements on the readme, which is the main entry point.

The part of the semver and LTS description is not related with installation but more on the projet itself, that is why i move this part
(Feel free to close if done on purpose)

Also, should i rewrite the numbers for the links?
Cheers

### CODE REVIEW

**Comment by Arman-Hosseini** [comment]
It was done on purpose.

File: README.md [hunk 1]
```diff
@@ -7,13 +7,14 @@
 BlaBlaCar.com and Spotify.com) and most of the [popular PHP projects][2] (including
 Drupal and Magento).
 
+Symfony follows the [semantic versioning][5] strictly, publishes "Long Term
+Support" (LTS) versions and has a [release process][6] that is predictable and
+business-friendly.
```
Comment (fabpot) [inline README.md]: I would instead remove this paragraph as we don't want to highlight the LTS.

**Review by fabpot — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (noniagriconomie) [reply README.md]: @fabpot hello, thanks for your comment

according to https://symfony.com/what-is-symfony

```
Symfony Philosophy
Embracing and promoting professionalism, best practices, standardization and interoperability of applications.
```

uppon the `professionalism` word
imho it is important to mention on the readme of such project :)
so may i ask you why LTS is going to be removed?

cheers

Comment (fabpot) [reply README.md]: Relying on LTS means that you only upgrade every 2 years. That's a LOT of work. Upgrading every 6 months is much easier. Now that Symfony is becoming quite good at not breaking BC, continuously upgrading seems a much better strategy to me.

Comment (noniagriconomie) [reply README.md]: Oki understood
I will continue this soon

**Pull request closed without merging**